benchmark/profile_throughput.py

```diff
@@ -307,6 +307,10 @@ def parse_args():
     # pytorch engine args
     pt_group = parser.add_argument_group('PyTorch engine arguments')
     ArgumentHelper.eager_mode(pt_group)
+    ArgumentHelper.dllm_block_length(pt_group)
```

Comment:
what's "dllm" short for?

Reply:
How about we suggest defining these parameters at model side?
At lmdeploy side, user can override the parameters through option `--hf-overrides`

Reply:
The name comes from https://arxiv.org/pdf/2506.13759 which means `Diffusion large language model`.
SDAR does not put the block length in model_config (I hope they did) so we have to expose the parameter here.